xbmc/network/WebServer.cpp

```diff
@@ -765,6 +766,10 @@ int CWebServer::CreateFileDownloadResponse(const std::shared_ptr<IHTTPRequestHan
   std::shared_ptr<XFILE::CFile> file = std::make_shared<XFILE::CFile>();
   std::string filePath = handler->GetResponseFile();
 
+  // white/black list access check
+  if (!CFileUtils::ZebraListAccessCheck(filePath))
+    return SendErrorResponse(request.connection, MHD_HTTP_NOT_FOUND, request.method);
```

Comment:
Why `MHD_HTTP_NOT_FOUND` (404) and not `MHD_HTTP_FORBIDDEN` (403) as the file exists (or we wouldn't make it until here)?